JobRunner/callback_server.py

```diff
@@ -29,27 +41,42 @@ def start_callback_server(ip, port, out_queue, in_queue, token, bypass_token):
         "KEEP_ALIVE_TIMEOUT": timeout,
         "REQUEST_MAX_SIZE": max_size_bytes,
     }
-    app.config.update(conf)
+    #app.config.update(conf)
+    config(conf)
     if os.environ.get("IN_CONTAINER"):
         ip = "0.0.0.0"
-    app.run(host=ip, port=port, debug=False, access_log=False)
+    #app.run(host=ip, port=port, debug=False, access_log=False)
+    uvconfig = uvicorn.Config("JobRunner.callback_server:app", host=ip, port=port,
+                              access_log=False, log_level="warning")
+    server = uvicorn.Server(uvconfig)
+    server.run()
```

Comment:
Need to see if the 
* monitor_jobrunner.py still finds these logs
* any missing settings we need